<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>600</width>
      <height>630</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>View3D</string>
     </attribute>
     <widget class="View3D" name="view3D" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>600</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>View2D</string>
     </attribute>
     <widget class="View2D" name="view2D" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>176</width>
     <height>41</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>Option</string>
     </property>
     <widget class="QRadioButton" name="radioButtonGPUTrace">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>GPU Raytracing</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonCPUTrace">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU Raytracing</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_shadow">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>141</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Use shadow</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_spot_light">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Spot light</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_point_light">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Point light</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_supersampling">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Supersampling</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_dir_light">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Directional light</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_multi_thread">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Multi-thread (CPU)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_use_kdtree">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>141</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Use KdTree</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="traceButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>440</y>
       <width>96</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Trace</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>480</y>
       <width>96</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>View2D</class>
   <extends>QWidget</extends>
   <header>view2d.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>View3D</class>
   <extends>QWidget</extends>
   <header>view3d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
